src/main/java/dev/vasyl/proj/dto/CreateBookRequestDto.java

```diff
@@ -8,14 +9,26 @@
 @Data
 public class CreateBookRequestDto {
     @NotNull
```

Comment:
use NotBlank for String

```suggestion
    @NotBlank
```